core/domain/classifier_domain.py

```diff
@@ -169,3 +170,162 @@ def validate(self):
             classifier_registry.Registry.get_classifier_by_algorithm_id(
                 self.algorithm_id))
         classifier_class.validate(self.cached_classifier_data)
+
+
+class TrainClassifierJob(object):
```

Comment:
I was thinking the same, TrainClassifierJob sounds weird. Thanks, will change it!